modules/ROOT/pages/box/box-connector-examples.adoc

```diff
@@ -5,7 +5,9 @@ endif::[]
 
 Try out the following examples using Studio or XML.
 
-== Anypoint Studio Example: Create Folder, Retrieve Folder, and Delete Folder Flow
+
+
+== Anypoint Studio Example: Folders
```

Comment:
So...it looks like I spent a ton of time editing the wrong thing.
Please take a look and apply those edits to the rest of this PR.
I have to move on to other editing requests.